globals.go

```diff
@@ -269,27 +272,32 @@ const (
 	InvalidDivideInputError    = "$divide only supports numeric types"
 )
 
-var InternalAliases []string = []string{
+var InternalAliases = []string{
 	CommitQueueAlias,
 	GithubPRAlias,
 	GithubChecksAlias,
 	GitTagAlias,
 }
 
-var TaskFailureStatuses []string = []string{
+// TaskNonGenericFailureStatuses represents some kind of specific abnormal failure mode.
+var TaskNonGenericFailureStatuses = []string{
 	TaskTimedOut,
-	TaskFailed,
 	TaskSystemFailed,
 	TaskTestTimedOut,
 	TaskSetupFailed,
 	TaskSystemUnresponse,
 	TaskSystemTimedOut,
 }
 
-var TaskUnstartedStatuses []string = []string{
+// TaskFailureStatuses represents all the ways that a task can fail, inclusive of system failures
+// and task failures.
+var TaskFailureStatuses = append(TaskNonGenericFailureStatuses, TaskFailed)
```

Comment:
I would avoid doing the append this way for the reason I mentioned in the previous comment.